vue/components/ui/molecules/search-persistent-local-storage/search-persistent-local-storage.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+    <div class="search-persistent-local-storage">
+        <global />
+        <search-persistent
+            v-bind:value.sync="valueData"
+            v-bind:filters.sync="filtersData"
+            v-bind:placeholder="placeholder"
+            v-bind:icon-visible="iconVisible"
+            v-bind:clear-visible="clearVisible"
+            v-bind:variant="variant"
+            v-bind:width="width"
+            v-bind:loading="loading"
+            v-on:update:filters="onUpdateFilters"

```

Comment:
Either use `sync` or listen to this event. You could use `sync` and then watch on `filtersData`, for instance.